app/components/markdown/markdown.tsx

```diff
@@ -288,6 +288,7 @@ const Markdown = ({
 
         return (
             <Hashtag
+                channelId={channelId}
```

Comment:
Sorry for my ignorance, but will this work properly in the search and recent mentions tabs?
Given the right context it should, but just confirming.

Reply:
yes, see https://github.com/mattermost/mattermost-mobile/blob/main/app/components/post_list/post/body/message/message.tsx#L88

But I'm going to double check the global threads

Reply:
ok, global threads do not have the hashtag tappable